README.md

```diff
@@ -1,35 +1,36 @@
 
 # CAP plugin for SAP Document Management Service
-The project is currently in the development stage, where our team is actively working on building and refining its core components to ensure it meets all specified requirements and objectives.
-
-The **@cap-java/sdm** package is [cds-plugin](https://cap.cloud.sap/docs/java/cds-plugins#cds-plugin-packages) that provides an easy CAP-level integration with [SAP Document Management Service](https://discovery-center.cloud.sap/serviceCatalog/document-management-service-integration-option). This package supports handling of attachments(documents) by using an aspect Attachments in SAP Document Management Service.  
+The **com.sap.cds:sdm** dependency is a [CAP Java plugin](https://cap.cloud.sap/docs/java/building-plugins) that provides an easy CAP-level integration with [SAP Document Management Service](https://discovery-center.cloud.sap/serviceCatalog/document-management-service-integration-option). This package supports handling of attachments(documents) by using an aspect Attachments in SAP Document Management Service.  
 This plugin can be consumed by the CAP application deployed on BTP to store their documents in the form of attachments in Document Management Repository.
 
 ## Key features
 
 - Create attachment : Provides the capability to upload new attachments.
+- Read attachment : Provides the capability to preview attachments.
+- Delete attachment : Provides the capability to remove attachments.
+- Rename attachment : Provides the capability to rename attachments.
 - Virus scanning : Provides the capability to support virus scan for virus scan enabled repositories.
 - Draft functionality : Provides the capability of working with draft attachments.
+- Display attachments specific to repository: Lists attachments contained in the repository that is configured with the CAP application.
 
 ## Table of Contents
 
 - [Pre-Requisites](#pre-requisites)
+- [Setup](#setup)
 - [Deploying and testing the application](#deploying-and-testing-the-application)
-- [Use @cap-java/sdm plugin](#use-cap-javasdm-plugin)
+- [Use com.sap.cds:sdm dependency](#use-the-comsapcdssdm-dependency)
 - [Known Restrictions](#known-restrictions)
 - [Support, Feedback, Contributing](#support-feedback-contributing)
 - [Code of Conduct](#code-of-conduct)
 - [Licensing](#licensing)
 
 ## Pre-Requisites
 * Java 17 or higher
-* CAP Development Kit (`npm install -g @sap/cds-dk`)
-* SAP Build WorkZone should be subscribed to view the HTML5Applications.
 * [MTAR builder](https://www.npmjs.com/package/mbt) (`npm install -g mbt`)
 * [Cloud Foundry CLI](https://docs.cloudfoundry.org/cf-cli/install-go-cli.html), Install cf-cli and run command `cf install-plugin multiapps`.
 
-> [!Note] **cds-services**
+> **cds-services**
 >
 > The behaviour of clicking attachment and opening it varies based on the version of cds-services used by the CAP application. 
```

Comment:
Instead of opening can the word be preview

Reply:
Also can we specify to add 3.4.0 preferably

Reply:
Done